experimental/SetPartitions/src/ColoredPartition.jl

```diff
@@ -0,0 +1,164 @@
+import Base.hash
+import Base.==
+import Base.copy
+
+"""
+ColoredPartition
+
+Initialize colored Partition object
+
+# Arguments
+- `partition`: Partition object which is generalized with coloring `color_upper_points` and `color_lower_points`
+- `color_upper_points`: color (in [0, 1]) of upper points
+- `color_lower_points`: color (in [0, 1]) of lower points
+"""
+struct ColoredPartition <: AbstractPartition
+    partition::Partition
+    color_upper_points::Array{Int64, 1}
+    color_lower_points::Array{Int64, 1}
```

Comment:
```suggestion
    color_upper_points::Vector{Int64}
    color_lower_points::Vector{Int64}
```
This should be adjusted everywhere else, too

Reply:
Also `Int` instead of `Int64`?